### ISSUE TITLE
Install Swift/T on JLSE KNL

### ISSUE THREAD

**Issue by jmjwozniak** [ECP-CANDLE/Supervisor#3]
with Lorenzo's Python.  Already half done...

**Comment by jmjwozniak** [comment]
Works- notes are here: http://swift-lang.github.io/swift-t/sites.html#_jlse_knl

**Comment by jmjwozniak** [comment]
Note that this uses my Python, not Lorenzo's.  Swift/T needs a Python shared object.

**Comment by llpesce** [comment]
I assume that I can put the Keras related libs outside of your tree. 
I will try and sort out how to make virtualenv friendly enough that you can move the libs from where I have them to your tree

**Comment by jmjwozniak** [comment]
Ok maybe make a separate ticket for that discussion.

**Comment by llpesce** [comment]
Epoch 19/20
800000/800000 [==============================] - 1277s - loss: 0.0907 - val_loss: 0.0891 - val_acc: 0.8336 - test_loss: 1.0966 - test_acc: 0.5668
Epoch 20/20
800000/800000 [==============================] - 1527s - loss: 0.0898 - val_loss: 0.0854 - val_acc: 0.8352 - test_loss: 1.1027 - test_acc: 0.5665


The P1B3 installation on JLSE that depends upon your python run is working.

File:
/home/lpesce/CANDLE/run/CobaltSrc/computeNode_sfw.sh
Described how to call it (the PYTHONPATH has the location of the files).

Something like
qsub -q knl_7210 -t $((60*24)) -n 1 /home/lpesce/CANDLE/run/CobaltSrc/computeNode.sh

Works for Cobalt. 

I am going to base my optimization runs on another tree. 




Lorenzo Pesce, Ph.D.
The Computation Institute
The University of Chicago, Northwestern University and  Argonne National laboratories
lpesce@uchicago.edu
